### PR TITLE
Release/1.4.0

### PR DESCRIPTION
**Pull request by sphuber**

### CODE REVIEW

**Comment by sphuber** [comment]
@ramirezfranciscof note that you should just review the last commit on top of this PR and make sure that it is complete with respect to the requirements for a release

**Review by ramirezfranciscof — APPROVED** [review]
So, I reviewed the changelog and added a couple of comments, but this is mostly up to the commiter criteria I guess, so feel free to take or leave any. I also checked the following:

- Version updated in `aiida/__init__.py` and `setup.json` (this was the only check from the wiki that I could see, the rest seem instructions on how to handle the github part, or did I miss something?).
- Headers added in: `aiida/orm/utils/repository.py`, ` aiida/repository/__init__.py`, `aiida/repository/common.py`, `tests/manage/external/test_rmq.py`, and `tests/static/__init__.py` (I'm guessing these are the only new files added?).
- There are some sections that Codecov seems to have detected without tests in `aiida/orm/utils/repository.py` and `aiida/repository/common.py`. Not sure what is up with that (these sections were not modified during the last commit, not sure why Codecov is analizing them here).

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
+- Add the `--paused` flag to `verdi process list` [[#4213]](https://github.com/aiidateam/aiida-core/pull/4213)
+- Make the loglevel of the daemonizer configurable [[#4276]](https://github.com/aiidateam/aiida-core/pull/4276)
+- `Transport`: add option to not use a login shell for all commands [[#4271]](https://github.com/aiidateam/aiida-core/pull/4271)
+- Implement `skip_orm` option for SqlAlchemy `Group.remove_nodes` [[#4214]](https://github.com/aiidateam/aiida-core/pull/4214)
+- `Dict`: allow setting attributes through setitem and `AttributeManager` [[#4351]](https://github.com/aiidateam/aiida-core/pull/4351)
+- `CalcJob`: allow nested target paths for `local_copy_list` [[#4373]](https://github.com/aiidateam/aiida-core/pull/4373)
+- `verdi export migrate`: add `--in-place` flag to migrate archive in place [[#4220]](https://github.com/aiidateam/aiida-core/pull/4220)
+
+### Bug fixes
+- `verdi`: make `--prepend-text` and `--append-text` options properly interactive [[#4318]](https://github.com/aiidateam/aiida-core/pull/4318)
```
[the remark this answers is not in the document]
Comment (sphuber) [reply CHANGELOG.md:23]: I can see how the `local_copy_list` can be considered a bug fix and happy to move it there. Sometimes, these are not a 100% clear and ultimately don't matter that much. Happy to move it to bug fix if you prefer. The `--prepend-text` though, I really think was a bug. It was always intended to be interactive but it wasn't.

**Comment by sphuber** [comment]
Thanks for the review @ramirezfranciscof . I have accepted the three suggestions of the change log.

**Pull request merged**